dataset_eval/evaluate.py

```diff
@@ -50,25 +51,27 @@ def evaluate_dataset(eval_job):
         db.dataset_eval.add_sets_to_job(eval_job["id"], train, test)
 
         logging.info("Generating filelist.yaml and copying low-level data for evaluation...")
-        filelist_path = os.path.join(temp_dir, "filelist.yaml")
-        filelist = dump_lowlevel_data(train.keys(), os.path.join(temp_dir, "data"))
+        filelist_path = os.path.join(eval_location, "filelist.yaml")
+        data_dir = os.path.join(eval_location, "data")
```

Comment:
Maybe it's better to keep it in `/temp`? In case script fails and it doesn't get to cleanup point.